romana-install/roles/stack/kubernetes/install-master/tasks/etcd_dl.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Had to do a double take to figure out that "etcd_dl" stands for "etcd_download". Could we just call it "etcd_download"?